socha-sdk/src/framework/sc/shared/DebugHint.kt

```diff
@@ -0,0 +1,26 @@
+package sc.shared
+
+import com.thoughtworks.xstream.annotations.XStreamAsAttribute
+
+/**
+ * An additional note to be appended to a move.
+ * These notes can then be displayed in the GUI if debugging is enabled.
+ *
+ * This enables faster debugging of clients and easier configuring of strategies as
+ * important information gets displayed together with the corresponding move (instead of the CLI)
+ * ===
+ * Ein Debughinweis ist ein Container für einen String, der einem Zug beigefuegt
+ * werden kann. Beigefuegte Debughints werden direkt in der grafischen
+ * Oberflaeche des Plugins angezeigt, wenn die Debugansicht gewaehlt wurde.
+ *
+ * Dies ermöglicht das schnellere Debuggen von Clients und besseres
+ * Konfigurieren von Strategien, denn es müssen keine Konsolenausgaben gesucht
+ * werden und die Hinweise werden immer zum passenden Zug angezeigt.
+ */
+class DebugHint(@XStreamAsAttribute val content: String = "") : Cloneable {
+    /**
+     * This constructs a hint of the form "key = value"
+     */
```

Comment:
I'd prefer this to be a one-line documentation comment. But end it with a dot for consistency - seems that some doc parsers care about that to create a summary or smth.

Reply:
Ah, yeah, of course
The original part had an overly extensive documentation - I shortened that but forgot to switch to multiline comment.

Reply:
But since I deleted that class in [d6a7961](https://github.com/CAU-Kiel-Tech-Inf/socha/pull/268/commits/d6a7961ac2b9210dc7d01103e74b69f6fc8feac7) anyways, unless we decide to add it back in, it doesn't matter